inputfiles/deprecatedMessage.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I don't see any automatic generator for this file. Is this file handcrafted?

Reply:
No, it is created by a script. It is why I change the .gitignore file ---- I did add the template file, for running the script by vscode quickly.
But sandersn think this file would not update frequently. So it could be maintained manually like others. You could see the previews for more info.